pkgs/top-level/all-packages.nix

```diff
@@ -15943,6 +15943,16 @@ in
     packages = [];
   };
 
+  rstudioServerWrapper = libsForQt5.callPackage ../development/r-modules/wrapper-rstudio-server.nix {
+    recommendedPackages = with rPackages; [
+      boot class cluster codetools foreign KernSmooth lattice MASS
+      Matrix mgcv nlme nnet rpart spatial survival
+    ];
+    # Override this attribute to register additional libraries.
+    packages = [];
+  };
```

Comment:
Personally, I prefer having the "default" attr being the one more likely to be used. If this will almost always be prefered over `rstudio-server`, then I would just make the packages be:
```
rstudio-server
rstudio-server-unwrapped
```
instead

But that's just my personal preference

Reply:
I think for consistency with the normal rstudio package we should keep this as is, but if we could get the rstudio maintainers on board I would be happy to switch

Reply:
in the interest of putting an initial version in, I am keeping this this way for now, would consider changing in the future